src/redux/message/message.js

```diff
@@ -0,0 +1,29 @@
+const GET_MESSAGE_API = 'messageReducer/GET_MESSAGE_API';
+
+const initialState = [];
+
+// const getMessage = (payload) => ();
+
+export const getMessageAPI = () => async (dispatch) => {
+  const res = await fetch('http://localhost:3000/greetings/show', {
+    mode: 'no-cors',
+  });
+  const data = await res.json();
```

Comment:
* I get this error in the console while I run the frontend of your app, Kindly fix this as no random greeting is being rendered 
![image](https://user-images.githubusercontent.com/52497246/211714734-9b264f94-8a31-4df1-9f5c-e1e14d54f403.png)

Reply:
the error was corrected and updated, I add the screenshot of my browser:

![image](https://user-images.githubusercontent.com/92118813/211874684-fae0fc93-2beb-4c74-8e37-9f53a280261c.png)